DQM/SiOuterTracker/python/OuterTrackerMonitorTTTrack_cfi.py

```diff
@@ -32,15 +32,15 @@
 #Phi of the track
     TH1_Track_Phi = cms.PSet(
         Nbinsx = cms.int32(60),
-        xmax = cms.double(math.pi),
-        xmin = cms.double(-math.pi)
+        xmax = cms.double(4.5),
```

Comment:
I can understand in the examples above to use 3.5 instead of pi in order to have a uniform binning, but 4.5? What's the reason behind?

Reply:
Ah, that was a mistake. It should be 3.5. Let me change that